rules/do-you-know-which-environments-you-need-to-provision-when-starting-a-new-project/rule.md

```diff
@@ -0,0 +1,49 @@
+---
+type: rule
+title: Do You Know Which Environments You Need To Provision When Starting A New
+  Project?
+uri: do-you-know-which-environments-you-need-to-provision-when-starting-a-new-project
+authors:
+  - title: Daniel Mackay
+    url: https://ssw.com.au/people/daniel-mackay/
+created: 2023-11-20T05:48:37.152Z
+guid: 8d165756-d35e-4d1a-8924-3c4f59523db4
+related:
+- ephemeral-environments
+- do-you-have-separate-development-testing-and-production-environments
+---
+Before any project can be used by a customer it must first be published to a production environment.  However, in order to provide a robust and uninterrupted service to customers, it is important that the production environment is not used for development or testing purposes.  To ensure this, we must setup a separate environment for each of these purposes.
+
+## What is each environment for?
+
+* **Production**: Real data being used by real customers
```

Comment:
Maybe add a few more characteristics of a production environment, like
Full-spec, security, redundancy, backups, etc.

Could move the last sentence from "Staging" and use it for "Production".

Reply:
```suggestion
* **Production**: Real data being used by real customers. This is the baseline/high watermark of all your environments. Lower environments will be lower spec, fewer redundancies, less frequent backups, etc. 
```

Reply:
Added a clarification to PROD as per the suggestion from @wicksipedia.